src/NoFrixion.MoneyMoov/Models/Beneficiary/BeneficiaryEvent.cs

```diff
@@ -24,11 +24,11 @@ public class BeneficiaryEvent
 
     public Guid BeneficiaryID { get; set; }
 
-    public BeneficiaryAuthoriseStatusEnum EventStatus { get; set; }
+    public string EventStatus { get; set; }
 
```

Comment:
Any reason to switch variable type to string?

Reply:
Yes. Because the event status will not be specific to authorise event. In an upcoming PR I am recording beneficiary updated event. The status will be "Updated" in that case.